### PR TITLE
cova api updated with srf_id

### PR DESCRIPTION
**Pull request by egovmis1**
@rushang7-eGov @rakesh-wt-egov @rohit-eGov @ramakrishnaippili-eGov

### CODE REVIEW

File: xstate-chatbot/nodejs/src/machine/vitals.js [hunk 1]
```diff
@@ -348,6 +348,7 @@ const vitalsFlow = {
                 context.slots.person.com_status = context.intention.com_status;
                 context.slots.person.fateh_kit_delivered = context.intention.fateh_kit_delivered;
                 context.slots.person.data_source_type = context.intention.data_source_type;
+                context.slots.person.srfC_id=context.intention.srf_Id;
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/vitals.js:351]: Does this work properly? 

`context.intention` is a string at most of the places.

Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js:351]: i have tested with cova and data is getting stored in cova reports after complete flow

Comment (rushang7-eGov) [reply xstate-chatbot/nodejs/src/machine/vitals.js:351]: Still, I suggest don't store anything in context.intention. It is only for storing intention which is a string. To implement this change you would have to make changes where you are storing values to context.intention.

Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js:351]: ok

Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js]: code changes committed

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**